Makefile

```
@@ -1,7 +1,11 @@
+tag = $$(git log -1 --format=%h)
```

- In shell scripts, global variable names conventionally use CAPITAL LETTERS.
- We can't have a space between the name and value when defining a variable.
- The $$ variable can't be used in a Makefile (at least it doesn't work on my machine).
- Instead, use the shell function as described here: https://www.gnu.org/software/make/manual/html_node/Shell-Function.html
- also need a new line after defining variables
```suggestion
TAG=$(shell git log -1 --format=%h)

```